commit/chainfee/outcome.go

```diff
@@ -65,9 +65,13 @@ func (p *processor) Outcome(
 			DataAvFeePriceUSD:    mathslib.CalculateUsdPerUnitGas(feeComp.DataAvailabilityFee, usdPerFeeToken.Int),
 		}
 
+		p.lggr.Warnw("Outcome chainFeeUsd", "chainFeeUsd", chainFeeUsd, "chain", chain, "feeComp", feeComp, "usdPerFeeToken", usdPerFeeToken)
```

Comment:
why warn?

Reply:
This PR is here only for testing purposes.